roles/mistral/defaults/main.yml

```diff
@@ -1 +1,4 @@
 mistral_version: 0.13
+mistral_db_password: StackStorm
+mistral_db_server: localhost
```

Comment:
Are you using external db (not `localhost`)?

Because `ANXS.postgresql` dependency https://github.com/StackStorm/ansible-st2/blob/bde33777da7b2327bc0571418bf720ce43f7f716/roles/mistral/meta/main.yml#L15-L18 will install/configure PostgreSQL on localhost.